core/background/scrobblers/baseScrobbler.js

```diff
@@ -155,11 +155,11 @@ define([
 					} else {
 						cb(response.session);
 					}
-				})
+				}.bind(this))
```

Comment:
You also can you arrow functions instead of binding `this`. Arrow functions have no own context, so the proper `this` is always available inside them.